package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.3.0",
+  "version": "0.4.0",
```

Comment:
When we release, the script will auto-bump the version, so you don't have to (and shouldn't) bump it in your PR.